src/charm.py

```diff
@@ -60,6 +86,13 @@ def _on_content_cache_pebble_ready(self, event) -> None:
         self.model.unit.status = MaintenanceStatus(msg)
         self.on.config_changed.emit()
 
+    def _on_nginx_prometheus_exporter_pebble_ready(self, event) -> None:
```

Comment:
Can you add type hints for the `event`?, also, this arg is missing in the docstring